modules/fts/pages/fts-secure-fts-queries.adoc

```diff
@@ -0,0 +1,44 @@
+= Secure Queries Using SSL
```

Comment:
Was this supposed to be a part of this PR? IIRC, it is not as per the doc writers..

Reply:
Will be included in the next PR.